cypress/integration/NewEvent/index.js

```diff
@@ -0,0 +1,177 @@
+        .blur();
+    cy.get('[data-test="dhis2-capture-form-field-w75KJ2mc4zz"]')
+        .find('input')
+        .type('Marcus');
+    cy.get('[data-test="dhis2-capture-form-field-zDhUuAYrxNC"]')
+        .find('input')
+        .type('Barnes');
+    cy.get('[data-test="dhis2-capture-form-field-cejWyOfXge6"]')
+        .find('input')
+        .type('Male{enter}', { force: true });
+});
+
+When('you submit the registration form', () => {
+    cy.get('[data-test="dhis2-capture-create-and-link-button"]')
+        .click();
+});
+
+When('you submit the event form with the associated relationship to the newly created person', () => {
+    cy.server();
+    cy.route('POST', '**/events').as('postEvent');
+    cy.route('POST', '**/trackedEntityInstances').as('postTrackedEntityInstance');
+    cy.route('POST', '**/relationships').as('postRelationship');
+    cy.get('[data-test="dhis2-uicore-splitbutton-button"]')
+        .click();
+});
+
+Then('the data should be sent to the server successfully', () => {
+    cy.wait('@postTrackedEntityInstance', { timeout: 30000 }).should('have.property', 'status', 200);
+    cy.wait('@postEvent', { timeout: 20000 }).should('have.property', 'status', 200);
+    cy.wait('@postRelationship', { timeout: 20000 }).should('have.property', 'status', 200);
+
+    // clean up
+    cy.get('@postRelationship').then((result) => {
+        const id = result.response.body.response.importSummaries[0].reference;
+        cy.buildUrl(Cypress.env('dhis2_base_url'), `api/relationships/${id}`)
+            .then((relationshipUrl) => {
+                cy.request('DELETE', relationshipUrl);
+            });
+    });
+    cy.get('@postTrackedEntityInstance').then((result) => {
+        const id = result.response.body.response.importSummaries[0].reference;
+        cy.buildUrl(Cypress.env('dhis2_base_url'), `api/trackedEntityInstances/${id}`)
+            .then((trackedEntityInstanceUrl) => {
+                cy.request('DELETE', trackedEntityInstanceUrl);
+            });
+    });
+    cy.get('@postEvent').then((result) => {
+        const id = result.response.body.response.importSummaries[0].reference;
+        cy.buildUrl(Cypress.env('dhis2_base_url'), `api/events/${id}`)
+            .then((eventUrl) => {
+                cy.request('DELETE', eventUrl);
+            });
+    });
+});
+
+When('you navigate to find a person relationship', () => {
+    cy.get('[data-test="dhis2-capture-add-relationship-button"]')
+        .click();
+    cy.get('[data-test="dhis2-capture-relationship-type-selector-button-mxZDvSZYxlw"]')
+        .click();
+    cy.get('[data-test="dhis2-capture-find-relationship-button"]')
+        .click();
+});
+
+When('you search for an existing unique id and link to the person', () => {
+    cy.get('[data-test="dhis2-capture-form-field-lZGmxYbs97q"]')
+        .find('input')
+        .type('9191132445122')
+        .blur(); // TODO: Look into why the click below is failing if the field is not blurred first
```

Comment:
yeah I had the same before. I assumed back then that has to do with some action not being triggered. 

But is a very good point

Reply:
Good catch :thu